cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,63 @@
 describe('Web Tables page', () => {
-  it('', () => {
+  const user = {
+    firstName: 'John',
+    lastName: 'Doe',
+    Email: 'JohnDoe@qa.team',
```

Comment:
Change the 'Email' key to 'email' in the user object for consistency.

Reply:
done